src/features/create-dao/CreateDAOForm.tsx

```diff
@@ -0,0 +1,42 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Style Imports
+import styles from './CreateDAOForm.module.scss';
+
+//- Component Imports
+import { Wizard } from '@zero-tech/zui/components/Wizard/Wizard';
+import { CreateDAOBody } from './CreateDAOBody';
+import { CreateDAOHeader } from './CreateDAOHeader';
+import { ConnectWallet } from '../ui/ConnectWallet';
+
+// Lib Imports
+import { useWeb3 } from '../../lib/hooks/useWeb3';
+
+//- Context Imports
+import { CreateDAOFormContextProvider } from './CreateDAOFormContext';
+
+export type CreateDAOFormProps = {
+	domainName: string;
+	onClose: () => void;
+};
+
+export const CreateDAOForm: FC<CreateDAOFormProps> = ({
+	domainName,
+	onClose,
+}) => {
+	const { account } = useWeb3();
+
+	const content = account ? (
+		<CreateDAOFormContextProvider>
+			<Wizard.Container className={styles.Container}>
+				<CreateDAOHeader subtitle={domainName} onClose={onClose} />
```

Comment:
You should be able to remove `CreateDAOHeader`. 

`WizardContainer` has the header built in. You could move the context from `CreateDAOHeader` up to `CreateDAOForm` and do something like...
```
<Wizard.Container className={styles.Container} header={title}  headerInfo='...' subHeader='...'>
```

From memory I think we still need to add StepBar to the wizard so maybe it is worth leaving this as is for now if the step bar is included in designs.

Reply:
Sounds good. Let me know once the StepBar is included in the wizard and I'll refactor!